src/main/java/edu/ucla/library/bucketeer/Item.java

```diff
@@ -161,7 +164,9 @@ public Item setAccessURL(final String aAccessURL) {
         final String filePath;
         final File file;
 
-        if (!hasFile() || myFilePath.isEmpty()) {
+        if (myFilePath == null) {
```

Comment:
Why not `if (myFilePath == null || !hasFile() || myFilePath.isEmpty()) {` but I'm thinking another way to handle this is to just do `myFilePath = Optional.empty()` in the default empty `public Item()` constructor. That would also address the issue with `getFilePath` returning a null instead of an empty optional, which I think is counter to what one would expect for an optional return type.